src/main/webapp/submit-recipe.html

```diff
@@ -29,7 +29,7 @@
           oninput="renderRecipe()"
         ></textarea>
         <br />
-        <input type="button" onclick="submitRecipe()" value="Submit"></input>
```

Comment:
This is meant to be an input of type button, not a button. This is because a button's default type is submit (which does the form action), and I explicitly want it to be submitted through `submitRecipe` and not the form action.

Reply:
I think the linter prefers you do something like <button type="button" ...>   then. For some reason it doesn't like input of type "button." I can revert it to input if you prefer though

Reply:
I believe `<button type="button">` works as well, so yeah go ahead with that

Reply:
Will do!